src/services/item/plugins/itemFlag/index.ts

```diff
@@ -20,25 +21,21 @@ const plugin: FastifyPluginAsync = async (fastify) => {
   fastify.addSchema(common);
 
   // get flags
-  fastify.get(
-    '/flags',
-    { schema: getFlags, preHandler: optionalIsAuthenticated },
-    async ({ user }) => {
-      return itemFlagService.getAllFlags(user?.account, buildRepositories());
-    },
-  );
+  fastify.get('/flags', { schema: getFlags }, async () => {
```

Comment:
I see you removed the requirement for authentication, what is the plan for this endpoint ?

Reply:
Mhmmm... okay I thought this was returning all the elements that were flagged, But in reality it just returns the static array of flag types. Maybe adding a small comment would help disambiguate ? In fine we should probably change the name of the endpoint, maybe `/flagTypes` ?

Reply:
I agree ! I'd name that route `GET /flags/types` I've added this route to https://github.com/graasp/graasp/issues/1288